presto-parquet/src/main/java/com/facebook/presto/parquet/writer/ParquetWriterOptions.java

```diff
@@ -47,10 +50,16 @@ public int getMaxPageSize()
         return maxPageSize;
     }
 
+    public int getMaxDictionaryPageSize()
+    {
+        return maxDictionaryPageSize;
+    }
+
     public static class Builder
     {
         private DataSize maxBlockSize = DEFAULT_MAX_ROW_GROUP_SIZE;
         private DataSize maxPageSize = DEFAULT_MAX_PAGE_SIZE;
+        private DataSize maxDictionaryPageSize = DEFAULT_MAX_DICTIONARY_PAGE_SIZE;
```

Comment:
maybe just set this to `DEFAULT_MAX_PAGE_SIZE` and leave a comment as to why the defaults are the same value?

```suggestion
        private DataSize maxDictionaryPageSize = DEFAULT_MAX_PAGE_SIZE;
```